cmd/create.go

```diff
@@ -71,27 +72,59 @@ func init() {
 	//flagApplyAnnotations(createCmd, "name", "__kubicorn_parse_list")
 	flagApplyAnnotations(createCmd, "profile", "__kubicorn_parse_profiles")
 
+	RootCmd.SetUsageTemplate(usageTemplate)
 	RootCmd.AddCommand(createCmd)
 }
 
 type profileFunc func(name string) *cluster.Cluster
 
-var alias = map[string]profileFunc{
-	"amazon":       profiles.NewSimpleAmazonCluster,
-	"aws":          profiles.NewSimpleAmazonCluster,
-	"do":           profiles.NewSimpleDigitalOceanCluster,
-	"digitalocean": profiles.NewSimpleDigitalOceanCluster,
-	"do-centos":    profiles.CentosDigitalOceanCluster,
-	"aws-centos":   profiles.CentosAmazonCluster,
+type profileMap struct {
+	profileFunc profileFunc
+	description string
+}
+
+var profileMapIndexed = map[string]profileMap{
```

Comment:
I agree - I think we need a map that maps items from 2 lists together. Can we follow up PR for now and get this in as is?